packages/core/src/connectors/index.test.ts

```diff
@@ -120,13 +120,15 @@ describe('getSocialConnectorInstanceById', () => {
 
 describe('getConnectorInstanceByType', () => {
```

Comment:
why not fix the test description here as this `getConnectorInstanceByType` function is renamed globally?